recipes/starshot_accessibility/recipe.yml

```diff
@@ -0,0 +1,24 @@
+name: Accessibility
+type: Starshot
+description: 'Sets up accessibility tools and functionality.'
+recipes:
+  - starshot_admin_theme
+install:
+  - a11y_form_helpers
+  - block_aria_landmark_roles
+  - editoria11y
+config:
+  actions:
+    gin.settings:
+      simple_config_update:
+        high_contrast_mode: true
+        accent_color: ''
+        focus_color: ''
+    editoria11y.settings:
+      simple_config_update:
+        ignore_elements: '.leaflet-map-pane *'
+    block.block.olivero_welcome_to_starshot:
```

Comment:
The block config yml file is in recipes/starshot/config and just importing on installation, but I'll remove the tweak here adding a role of complimentary was more about showing off that it can be done, rather than it needing to be done. I'll double-check on the value of it and add it back if it is a more solid requirement.